docs/docs/integrations/platforms/anthropic.mdx

```diff
@@ -1,43 +1,44 @@
 # Anthropic
 
-All functionality related to Anthropic models.
+>[Anthropic](https://www.anthropic.com/) is an AI safety and research company, and is the creator of `Claude`.
+This page covers all integrations between `Anthropic` models and `LangChain`.
 
-[Anthropic](https://www.anthropic.com/) is an AI safety and research company, and is the creator of Claude.
-This page covers all integrations between Anthropic models and LangChain.
+## Installation and Setup
 
-## Installation
+To use `Anthropic` models, you need to install a python package:
 
-To use Anthropic models, you will need to install the `langchain-anthropic` package.
-You can do this with the following command:
-
-```
-pip install langchain-anthropic
+```bash
+pip install -U langchain-anthropic
 ```
 
-## Environment Setup
-
-To use Anthropic models, you will need to set the `ANTHROPIC_API_KEY` environment variable.
+You need to set the `ANTHROPIC_API_KEY` environment variable.
 You can get an Anthropic API key [here](https://console.anthropic.com/settings/keys)
 
-## `ChatAnthropic`
+## LLMs
 
-`ChatAnthropic` is a subclass of LangChain's `ChatModel`.
-You can import this wrapper with the following code:
+### [Legacy] AnthropicLLM
 
-```
-from langchain_anthropic import ChatAnthropic
-model = ChatAnthropic(model='claude-3-opus-20240229')
+**NOTE**: `AnthropicLLM` only supports legacy `Claude 2` models. 
+To use the newest `Claude 3` models, please use `ChatAnthropic` instead.
+
+See a [usage example](/docs/integrations/llms/anthropic).
+
+```python
+from langchain_anthropic import AnthropicLLM
+
+model = AnthropicLLM(model='claude-2.1')
 ```
 
-Read more in the [ChatAnthropic documentation](/docs/integrations/chat/anthropic).
+## Chat Models
 
-## [Legacy] `AnthropicLLM`
+### ChatAnthropic
```

Comment:
The chat model shold probably be surfaced prior to the legacy documentation?

Reply:
@eyurtsev I've added a note that ChatAnthropic should be used instead of the deprecated AnthropicLLM.
Do you mean, that we need to add `### ChatAnthropic` section under the `## LLMs` section by copy-past it from `## Chat models` section?